tests/acceptance/smoke-test-slow.js

```diff
@@ -87,6 +87,15 @@ describe('Acceptance: smoke-test', function() {
     expect(output).to.include('***CUSTOM_TESTEM_JS**');
   }));
 
+  it('eslint passes after running ember new', co.wrap(function *() {
+    let result = yield runCommand(path.join('.', 'node_modules', '.bin', 'eslint'), appRoot);
```

Comment:
@Gaurav0 this test seems to be failing on AppVeyor because the `eslint` bash script can only be run on unix, but not on windows. can you take a look at that?

Reply:
How does one run ESLint on Windows?

Reply:
I think on Windows it puts a eslint.cmd in the .bin folder